tests/test-maximize.lua

```diff
@@ -41,6 +41,22 @@ local steps = {
         c.maximized_horizontal = false
         return true
     end,
+
+    -- Test restoring client.border_width
+    function()
+        local c = client.get()[1]
+        local test_width = 2
```

Comment:
How about making this `c.border_width * 2` so that the test also works if the default border width is changed to 2?